website/content/gateway/api_reference/resources/authorizations/authorizations_methods/token.md

```diff
@@ -10,3 +10,97 @@ Two token frameworks are supported:
 
 * `token[m4m]`: Mastercard Digital Enablement Service (MDES) for Merchants (M4M)
 * `token[vts]`: Visa Token Service (VTS)
+
+Requirement of some parameters depends on the initiator of the transaction; a parameter might be required for cardholder-initiated transactions (CITs) and otherwise optional. See the details for each parameter.
+
+{{% notice %}}
+**Notice**: Signing is required to use the `token` payment methods.
+{{% /notice %}}
+
+#### Method: token[m4m]
+
+The required values are found in either the MDES `transact` response or the Secure Card on File (SCOF) `checkout` response.
+
+{{% description_list %}}
+{{% description_term %}}token[m4m][tan] {{% regex %}}[0-9]{12,19}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Token Account Number (TAN) of the token to charge.
+
+Found in `encryptedPayload.encryptedData.accountNumber` (MDES) or `encryptedPayload.token.paymentToken` (SCOF).
+{{% /description_details %}}
+
+{{% description_term %}}token[m4m][expire_month] {{% regex %}}[0-9]{2}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Expiry month of token to charge.
+
+Found in `encryptedPayload.encryptedData.applicationExpiryDate` (MDES) or `encryptedPayload.token.tokenExpirationMonth` (SCOF).
+{{% /description_details %}}
+
+{{% description_term %}}token[m4m][expire_year] {{% regex %}}20[0-9]{2}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Expiry year of token to charge.
+
+Found in `encryptedPayload.encryptedData.applicationExpiryDate` (MDES) or `encryptedPayload.token.tokenExpirationYear` (SCOF).
+{{% /description_details %}}
+
+{{% description_term %}}token[m4m][eci] {{% regex %}}0[267]{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Electronic Commerce Indicator.
+
+Found in `eci` (SCOF). (Not available in MDES.)
+{{% regex_optional %}}Required for CITs; otherwise optional (defaults to `07`).{{% /regex_optional %}}
+{{% /description_details %}}
+
+{{% description_term %}}token[m4m][tav] {{% regex %}}[:base64:]{28}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Token Authentication Value (TAV). Also known as token cryptogram and Digital Secure Remote Payments (DSRP) cryptogram.
+
+Must start with `[A-P]` to be a DSRP cryptogram.
```

Comment:
```suggestion
Only DSRP cryptograms are supported. The value must start with `[A-P]` to be a DSRP cryptogram.
```

Reply:
:+1: More clear given the above sentence.